.prettierignore

```diff
@@ -0,0 +1,10 @@
+**/build
+**/node_modules
+**/lib
+**/dist
+**/package.json
```

Comment:
Why are the package.json files ignored?

Reply:
@vidartf I also took the same default config of JupyterLab. I will change that if you prefer.